src/ExpressiveInstaller/OptionalPackages.php

```diff
@@ -42,27 +42,27 @@ class OptionalPackages
     /**
      * @var array
      */
-    private static $config;
+    public static $config;
```

Comment:
Those are not being tested. They are public so I can set them from within a test. Doing it this way I don't need to repeat the code to setup the OptionalPackages class.